src/third_party/catch/include/helpers.hpp

```diff
@@ -103,7 +103,10 @@
     auto collection_count_with_opts = libmongoc::collection_count_with_opts.create_instance();    \
     auto collection_create_index = libmongoc::collection_create_index.create_instance();          \
     auto collection_find = libmongoc::collection_find.create_instance();                          \
-    auto collection_aggregate = libmongoc::collection_aggregate.create_instance();
+    auto collection_aggregate = libmongoc::collection_aggregate.create_instance();                \
+    auto collection_get_name = libmongoc::collection_get_name.create_instance();                  \
+    collection_get_name->interpose([](mongoc_collection_t*){ return "dummy_collection"; });       \
+    auto collection_rename = libmongoc::collection_rename.create_instance();
```

Comment:
That would be good indeed. But at this point I am afraid my boss told me not to use too much time on this PR any more.